node/testing.go

```diff
@@ -42,6 +42,9 @@ func TestNode(t *testing.T, tp Type, opts ...Option) *Node {
 		WithNetwork(params.Private),
 		WithRPCPort("0"),
 		WithKeyringSigner(TestKeyringSigner(t)),
+		func(sets *settings) { // override the block time
+			sets.cfg.Services.BlockTime = time.Microsecond
+		},
```

Comment:
Couldn't this be pulled out as a `WithBlockTime` in `config_opts.go`?

Reply:
Originally I had this option available in `config_opts.go` but we shouldn't really expose this as an option configurable to users. We mainly need to override the default network block time for testing purposes. Having it as a config value in general is a bit iffy because if we do end up using this value somewhere else in the codebase, we have to be very careful how it's passed in / make sure there are no discrepancies, etc.